### DISCUSSION TITLE
Information about 'Skipped' and 'NotRun' tests

### DISCUSSION THREAD

**Discussion by magnusjak** [maester365/maester#276] [Ideas]
Hi, and thanks for a very nice product!

I might have overlooked something, but I can't seem to find any information about why some of my tests are being 'Skipped' and 'NotRun'. Could the reason for this be added in the output? At first I thought that it might be permission or service issues, but I believe that I have done these things correctly.
This is an example of a test that was skipped for me:
![image](https://github.com/maester365/maester/assets/103104630/87c4bed8-9a72-45a0-aee2-c9e49901504f)

Having some information that says something about this would help a lot! Examples of reasons I can think of:
1. Not enough permissions to perform this test, permission/role X is needed.
2. Module X is not installed.
3. Could not connect to Exchange/Graph/Azure.
4. Setting X is missing, therefore this test that is dependent on that setting can't be run

**Comment by soulemike** [comment]
Thanks for the feedback, and it is definitely being worked on. #269 shows a preview of adding more context to these skips.
